velox/functions/lib/tests/Re2FunctionsTest.cpp

```diff
@@ -56,12 +56,34 @@ class Re2FunctionsTest : public test::FunctionBaseTest {
   }
 
  protected:
+  const char* alphaNumCharSet_ =
+      "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+  const char* wildcardCharSet_ = "_%";
+  const char* singleWildcardCharSet_ = "_";
+  const char* anyWildcardCharSet_ = "%";
+
   template <typename T>
   void testRe2ExtractAll(
       const std::vector<std::optional<std::string>>& inputs,
       const std::vector<std::optional<std::string>>& patterns,
       const std::vector<std::optional<T>>& groupIds,
       const std::vector<std::optional<std::vector<std::string>>>& output);
+
+  std::string generateRandomString(
+      const char* charArr,
+      std::optional<vector_size_t> minLen = 1,
+      std::optional<vector_size_t> maxLen = 10) {
+    vector_size_t arrLen = strlen(charArr);
+    vector_size_t maxVal = maxLen ? *maxLen : 10;
+    vector_size_t minVal = minLen ? *minLen : 1;
+    vector_size_t len = rand() % maxVal + minVal;
```

Comment:
Tests using rand() may fail occasionally and failures may be hard to reproduce. Better write deterministic tests.

Reply:
Sure, thank you! The benchmark still uses randomly generated strings though since it seems appropriate there, please let me know if it should be removed from there as well.